### PR TITLE
Expose the alternate identifiers in RDF

### PR DESCRIPTION
**Pull request by magopian**
Fixes [#1534](https://github.com/datagouv/data.gouv.fr/issues/1534)

### CODE REVIEW

File: udata/core/dataset/rdf.py [hunk 1]
```diff
@@ -200,16 +201,25 @@ def dataset_to_rdf(dataset, graph=None):
...
     id = dataset_to_graph_id(dataset)
 
+    graph = graph or Graph(namespace_manager=namespace_manager)
+    d = graph.resource(id)
+
     # Expose upstream identifier if present
     if dataset.harvest and dataset.harvest.dct_identifier:
-        identifier = dataset.harvest.dct_identifier
+        d.set(DCT.identifier, Literal(dataset.harvest.dct_identifier))
+        alternate_identifier = URIRef(
```
Comment (maudetes) [inline udata/core/dataset/rdf.py:210]: Shouldn't the identifier be a literal? because else it may be a resource described in the graph.

Comment (magopian) [reply udata/core/dataset/rdf.py:210]: I'm confused, didn't we decide to use [URIRef for the `endpointURL`](https://github.com/opendatateam/udata/pull/3179/files#r1821117993)? Is this any different?

I'm happy to change it, I would just love to understand the logics of when to use a Literal (simple string, right?) or a URIRef (I thought `ref` in `URIRef` meant "reference to another resource described in the graph", but then we used it for the `endpointURL` which is just a URL, not another resource described in the graph).

Comment (maudetes) [reply udata/core/dataset/rdf.py:210]: I'm not 100% sure about the logic either.
I think that endpointURL is an identified resource on the web whereas an identifier is an identifying string?
However, we can base our decision on:
* the spec, which shows [endpoint URL](https://www.w3.org/TR/vocab-dcat-3/#Property:data_service_endpoint_url) in range Resource and [identifier having a notation](https://semiceu.github.io/DCAT-AP/releases/3.0.0/#Identifier) in range Literal
* usage in real life examples, ex: [this dataset on data.europa.eu](https://data.europa.eu/data/datasets/34fc887b-be29-47bf-a2b3-67428d47d434?locale=en) (with both adms:identifier and dcat:endpointUrl)

File: udata/core/dataset/rdf.py [hunk 2]
```diff
@@ -207,15 +207,20 @@ def dataset_to_rdf(dataset, graph=None):
...
             endpoint_for(
                 "datasets.show_redirect",
                 "api.dataset",
                 dataset=dataset.id,
                 _external=True,
             )
         )
-        d.add(ADMS.identifier, alternate_identifier)
+        alt.set(RDF.type, ADMS.Identifier)
+        alt.set(DCT.creator, Literal("https://data.gouv.fr"))
```
Comment (maudetes) [inline udata/core/dataset/rdf.py:221]: I think we should use `SITE_TITLE` config

**Review by maudetes — APPROVED** [review]
I think it works nicely, thank you!

**Pull request merged**